### ISSUE TITLE
Gerbera UI - Create Walkthrough Documentation

### ISSUE THREAD

**Issue by elmodaddyb** [gerbera/gerbera#245]
The new Gerbera UI is the default UI.

**Before we** release Gerbera UI to public
**We should** document the UI changes as a walkthrough
**So that** users have an introduction to the new UI

**Comment by elmodaddyb** [comment]
I am writing an automated script which does the following:

1. Automatically generates screenshots of the application
2. Sizes images automatically
3. Stores images in `/doc/` for publishing

Other ideas and approaches welcome!

**Comment by elmodaddyb** [comment]
I completed the automated capture/resize of screenshots of the gerbera UI walkthrough.  I will write the sphinx documentation soon.

> Never used sphinx before, so need to learn a bit.

**Comment by elmodaddyb** [comment]
I continue to make good progress on the documentation.  The following documentation is converted:
- Gerbera UI
- Installing
- Compiling
- Running

Once complete this will encompass the existing `index.rst` and the `scripting.lyx` and `transcoding.lyx`.

The new documentation is using the Sphinx documentation build system!

**Comment by elmodaddyb** [comment]
I completed the UI documentation and converted remaining documentation items to **reStructuredText**  All existing **lyx** and **txt** and **md** have been consolidated into the `/doc` folder as **reStructuredText**.

- Daemon setup (launchd, systemd, solaris)
- Configuration
- Scripting
- Transcoding

All together the documentation site is nearly complete (_still need to wire up a Travis build_)

Here is what the documentation site looks like now!

![gerbera-doc](https://user-images.githubusercontent.com/8599799/35781218-fe6ed3bc-09b4-11e8-90ad-cd9399b91cfd.png)

Here is the build example
```
Build Gerbera Documentation

Running Sphinx v1.6.7
loading pickled environment... not yet created
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 15 source files that are out of date
updating environment: 15 added, 0 changed, 0 removed
reading sources... [100%] ui                                                                                                 
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] ui                                                                                                  
generating indices... genindex
writing additional pages... search
copying images... [100%] _static/task-message.png                                                                            
copying static files... done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded.

SUCCESS generated Gerbera documentation:

/development/gerbera/doc/_build/dist
```

**Comment by v00d00** [comment]
That looks so good! Oh my words.

**Comment by elmodaddyb** [comment]
I am going add a Travis Build step that builds documentation on `origin/master`.  I am not sure how the `/doc` directory is linked up to `docs.gerbera.io` to get it fully automated however.

**Comment by v00d00** [comment]
You don't need to do that, there is already a webhook, so just pushing to master should be enough.

**Comment by elmodaddyb** [comment]
@v00d00

If i understand the webhook correctly the `do_sphinx.sh` is not related, and is just a _nice-to-have_ for local development.

The webhook uses the `index.rst` file at the root of the directory and generates content based on that file, considering `conf.py` settings.

Is your understanding similar?  It will help confirm the upcoming *rst* only changes I have will work with the `webhook` as long as everything is wired into `index.rst`.

Thanks,

Eamonn

**Comment by v00d00** [comment]
@elmodaddyb that is my understanding too.

**Comment by elmodaddyb** [comment]
The changes to consolidate/convert documentation to **reStructuredText** and include in the new `docs.gerbera.io` site are complete.

Its a whopper of a PR, about 11K deltas, across 15 files(_all documentation_).  I will submit PR this weekend, after an additional editorial review.

I did not create a Travis CI build step, as this seems redundant given the `hook` setup for **readthedocs**.

**Comment by elmodaddyb** [comment]
Closed via #252
